apps/site/assets/css/_footer.scss

```diff
@@ -9,47 +9,122 @@
   background-color: $footer-color;
   color: $white;
   min-width: 20rem;
-  padding-bottom: $base-spacing * 2;
-  padding-top: $base-spacing * 2;
+  padding-bottom: 24px;
+  padding-top: 24px;
+  border-radius:  0 50px 0 0;
+  flex-basis: 70%;
 
   @include media-breakpoint-up(xxl) {
     font-size: $font-size-base;
   }
 
-  @include media-breakpoint-down(md) {
+  @include media-breakpoint-up(lg) {
+    height: 100%;
+    padding-left: 1rem;
+    padding-right: 120px;
+    position: relative;
+    p {
+      margin-left: 40px;
+    }
+  }
+
+  @include media-breakpoint-only(md) {
     font-size: $font-size-base-sm;
+    z-index: 1;
+    position: absolute;
+    width: 100%;
   }
 
-  @media print {
-    display: none;
+  .container {
+    padding-bottom: 44px;
+    padding: 0 4px 0 4px;
+
+    @include media-breakpoint-up(lg) {
+      padding-left: 0px;
+      padding-right: 4px;
+    }
+
+    .m-footer__group {
+      .m-footer__group-title {
+        white-space: nowrap;
+      }
+    }
   }
 }
 
-.m-footer__group {
-  @include media-breakpoint-up(md) {
-    @include make-col();
-    @include make-col-span(3);
-    padding-bottom: $base-spacing * 2;
+.m-footer__links {
+  display: grid;
+  gap: 1.75rem;
+  grid-template-columns: 1fr 1fr 1fr;
+
+  @include media-breakpoint-down(xs) {
+    display: block;
+
+  }
+
+  a {
+    color: $brand-primary-light-contrast;
+    display: block;
+    margin-bottom: .75rem;
+    font-size: 18px;
+    @include media-breakpoint-up(lg) {
+      margin-bottom: 1rem;
+    }
+
+    @include media-breakpoint-down(xs) {
+      margin-bottom: 4px;
+    }
+
+    &:hover {
+      color: $brand-primary-light;
+      text-decoration: underline;
+    }
+
+    &:active {
+      color: $brand-primary;
+      text-decoration: underline;
+    }
+
+    &:focus {
+      outline: 2px solid $brand-secondary;
+    }
   }
 }
 
-.m-footer__group-title {
-  @include h5();
-  font-size: 1.04em; // Tweak so that titles appear same size as links (~15px)
-  margin-bottom: .5rem; // Match vertical spacing of listed items below
+.m-footer__group {
+  margin-bottom: 1.75rem;
 }
 
-.m-footer__item {
-  display: block;
-  margin: .25rem 0;
+.m-footer__group-title {
+  @include h3();
+  margin: 0 0 24px;
 
-  a {
-    color: $brand-primary-light-contrast;
+  @include media-breakpoint-only(xs) {
+    font-size: 12px;
+  }
+
+  @include media-breakpoint-only(sm) {
+    font-size: 1.15rem;
+  }
+
+  @include media-breakpoint-up(lg) {
+    font-size: 16px;
+    line-height: 30px;
+    font-family: Inter UI,Helvetica Neue,Helvetica,Arial,sans-serif;
+    margin-bottom: 4px;
+    margin-top: 16px;
   }
 }
 
 .m-footer__copyright {
   font-size: $font-size-base-xs;
+  @include media-breakpoint-up(lg) {
+    position: absolute;
```

Comment:
**issue:** Using `position: absolute` takes the element out of the regular layout flow and positions it relative to the closest parent that has a `position: relative` value - it should generally be used sparingly. This and the `.lower-footer` also having `position: absolute`, combined with various `display: flex` areas, have me concerned that the `<footer>` layout is a bit more complicated than it needs to be.

**question:** can the flexbox powers already bestowed by the `.lower-footer` element be leveraged to position its containing elements?

Reply:
**issue:** Using `position: absolute` takes the element out of the regular layout flow and positions it relative to the closest parent that has a `position: relative` value - it should generally be used sparingly. This and the `.lower-footer` also having `position: absolute`, combined with various `display: flex` areas, have me concerned that the `<footer>` layout is a bit more complicated than it needs to be.

**question:** can the flexbox powers already bestowed by the `.lower-footer` element be leveraged to position its containing elements?